save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -69,6 +70,32 @@ internal class OrganizationController(
         organizationService.findByName(organizationName)?.avatar.let { ImageInfo(it) }
     }
 
+    /**
+     * @param organizationName name of an organization
+     * @param newCanCreateContests new value of flag Organization.canCreateContests
+     * @param authentication an [Authentication] representing an authenticated request
+     * @return response
+     */
+    @PostMapping("/{organizationName}/manage-contest-permission")
+    @Suppress("UnsafeCallOnNullableType")
+    fun changeCanCreateContests(
+        @PathVariable organizationName: String,
+        @RequestParam newCanCreateContests: Boolean,
+        authentication: Authentication
+    ): Mono<StringResponse> = Mono.justOrEmpty(
+        Optional.ofNullable(
+            organizationService.findByName(organizationName)
```

Comment:
we can search on next step
also you can use method 'toMono' instead of Optional

Reply:
We can, but it seems that there is no difference

Reply:
code is more accurate

Reply:
ok, will do it

Reply:
Done